doc/bipart.xml

```diff
@@ -433,29 +433,39 @@ the argument does not define a partial perm,]]></Example>
 
       If the optional second argument <A>n</A> is not present, then the maximum
       of the degree and codegree of <A>x</A> plus 1 is used by default. If the
-      second argument <A>n</A> is not greater than this maximum, then
-      <K>fail</K> is returned.
+      second argument <A>n</A> is not greater than this maximum, then an Error
```

Comment:
Please change to lower case "Error", and is an error not "given", or "then an error will occur." It isn't really returned.